_posts/2016/09/2016-09-09-community-call.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+authors: ["Tracy Teal and Kate Hertweck"]
+title: "September Carpentries Community Call"
+date: 2016-09-09
+time: "00:16:00"
+category: ["Community"]
+---
+
+Our next Carpentries Community Call (formerly called Lab Meeting or 
+Town Hall meeting) will be help Thursday, September 15. These meetings will now be 
+monthly on the third Thursday of every month. It would be great to 
+see instructors there! These calls are a great chance to connect with 
+other Carpentry instructors and get updates and information on 
+important and interesting topics for the community. 
+
```

Comment:
Potentially add call times here? 

Times: 7am Pacific / 10am Eastern / 2pm UTC / 12am (Sept 16th) Sydney
             and 
            4pm Pacific / 7pm Eastern / 11pm UTC / 9am (Sept 16th) Sydney

(formatting could be different)